bundles/org.eclipse.chronograph.api/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,7 @@
+eclipse.preferences.version=1
```

Comment:
Please copy the settings from Eclipse Passage sources

Reply:
Settings copied.